moto/timestreaminfluxdb/models.py

```diff
@@ -270,6 +286,255 @@ def untag_resource(self, resource_arn: str, tag_keys: List[str]) -> None:
+    ) -> Tuple[str, str, str, str, Dict[str, Any]]:
+        validate_name(name)
+
+        for param_group in getattr(self, "db_parameter_groups", {}).values():
+            if param_group["name"] == name:
+                raise ConflictException(
+                    f"A DB parameter group with the name {name} already exists"
+                )
+
+        if not hasattr(self, "db_parameter_groups"):
+            self.db_parameter_groups = {}
+
+        param_group_id = random_id()
+
+        arn = f"arn:aws:timestream-influxdb:{self.region_name}:{self.account_id}:db-parameter-group/{param_group_id}"
+
+        param_group = {
+            "id": param_group_id,
+            "name": name,
+            "arn": arn,
+            "description": description or "",
+            "parameters": parameters or {},
+        }
+
+        self.db_parameter_groups[param_group_id] = param_group
+
+        if tags:
+            self.tag_resource(arn, tags)
+
+        return (
+            param_group["id"],
+            param_group["name"],
+            param_group["arn"],
+            param_group["description"],
+            param_group["parameters"],
+        )
+
+    def get_db_parameter_group(
+        self, identifier: str
+    ) -> Tuple[str, str, str, str, Dict[str, Any]]:
+        if not hasattr(self, "db_parameter_groups"):
+            raise ResourceNotFoundException(
+                f"DB parameter group with identifier {identifier} not found"
+            )
+
+        param_group = self.db_parameter_groups.get(identifier)
+        if not param_group:
+            raise ResourceNotFoundException(
+                f"DB parameter group with identifier {identifier} not found"
+            )
+
+        return (
+            param_group["id"],
+            param_group["name"],
+            param_group["arn"],
+            param_group["description"],
+            param_group["parameters"],
+        )
+
+    @paginate(pagination_model=PAGINATION_MODEL)
+    def list_db_parameter_groups(self) -> List[Dict[str, str]]:
+        if not hasattr(self, "db_parameter_groups") or not self.db_parameter_groups:
+            return []
+
+        items = []
+        for param_group_id, param_group in self.db_parameter_groups.items():
+            items.append(
+                {
+                    "id": param_group["id"],
+                    "name": param_group["name"],
+                    "arn": param_group["arn"],
+                    "description": param_group["description"],
+                }
+            )
+
+        return items
+
+    @paginate(pagination_model=PAGINATION_MODEL)
+    def list_db_clusters(self) -> List[Dict[str, object]]:
+        if not hasattr(self, "db_clusters") or not self.db_clusters:
+            return []
+
+        items = []
+        for cluster_id, cluster in self.db_clusters.items():
+            if cluster.get("status") == "DELETED":
+                continue
+
+            items.append(
```

Comment:
As above - we could just return a `List[Cluster]` instead, making it a lot easier on the caller to understand what it's receiving